AppService.Acmebot/Functions/SharedActivity.cs

```diff
@@ -455,9 +461,34 @@ public Task<Certificate> UploadCertificate([ActivityTrigger] (Site, string, byte
         }
 
         [FunctionName(nameof(UpdateSiteBinding))]
-        public Task UpdateSiteBinding([ActivityTrigger] Site site)
+        public async Task UpdateSiteBinding([ActivityTrigger] Site site)
         {
-            return _webSiteManagementClient.WebApps.CreateOrUpdateAsync(site);
+            // _options
+            var Client = _webSiteManagementClient;
+            var builder = new UriBuilder(_environment.ResourceManager);
+            var path = "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}";
```

Comment:
Ah, sorry, should have mentioned the string interpolation support
````
 var x = "atohesu";
 $"/subscriptions/{x}/...."
```
support in `C#`.

Reply:
@sgryt i knew that too .... i just miss printf already